.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 8 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27036&commitId=f035af118368cefd907cde5558295ca466b88b71&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1884 screenshots unchanged, 0 screenshots with different dimensions and 8 screenshots with visible difference.


  <details><summary><b>unknown</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|1831483| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|1832333| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|1827930| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|1827931| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|1828108| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|1828114| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|1821426| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>